tests/poll/test_models.py

```diff
@@ -0,0 +1,163 @@
+from bulbs.poll.models import Poll, Answer
+from bulbs.utils.test  import (
+    BaseIndexableTestCase,
+    make_vcr,
+    random_title,
+    mock_vault,
+)
+
+from datetime import datetime
+import os
+import pytz
+import random
+import re
+import requests
+import requests_mock
+import simplejson as json
+
+from bulbs.utils import vault
+
+from .common import SECRETS
+
+vcr = make_vcr(__file__)  # Define vcr file path
+
+class PollTestCase(BaseIndexableTestCase):
+
+    """ Tests for the Poll model class """
+
+    @vcr.use_cassette()
+    @mock_vault(SECRETS)
```

Comment:
As future cleanup, I figured out how to have these mock_vaults apply at a class-level, so you only need to tag the entire class once.

https://github.com/theonion/combine/blob/master/combine/utils/tests.py#L76